python/lsst/analysis/tools/tasks/ccdVisitTableAnalysis.py

```diff
@@ -87,24 +87,3 @@ class CcdVisitTableAnalysisConfig(AnalysisBaseConfig, pipelineConnections=CcdVis
 class CcdVisitTableAnalysisTask(AnalysisPipelineTask):
     ConfigClass = CcdVisitTableAnalysisConfig
     _DefaultName = "ccdVisitTableAnalysisTask"
-
-    def runQuantum(self, butlerQC, inputRefs, outputRefs):
-        # Docs inherited from base class.
-        inputs = butlerQC.get(inputRefs)
-        dataId = butlerQC.quantum.dataId
-        plotInfo = self.parsePlotInfo(inputs, dataId)
-        data = self.loadData(inputs["data"])
-        skymap = None if "skymap" not in inputs.keys() else inputs["skymap"]
-        camera = None if "camera" not in inputs.keys() else inputs["camera"]
```

Comment:
So, if I read this correctly, we no longer need a special carve-out for `camera` here - is that correct? I see the base `runQuantum` has a carve-out for `skymap`, but not for `camera`.

Reply:
In this ticket, even the base class shouldn't have a special carve-out for `skymap`. Defining connections should be sufficient, and no specializion of the `runQuantum` method is required. This is the main motivation for this ticket, because it was causing much confusion among the SciUnit members.

Reply:
Should we therefore also remove the special carve out for `skymap` here too?

https://github.com/lsst/analysis_tools/blob/1ca23f87940ab22fe96e2fce7408de272e41e4df/python/lsst/analysis/tools/interfaces/_task.py#L446

Or is this needed elsewhere?

Reply:
The PR removes this. I made an inline comment just now.